### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Modules_Screen Reader: When we press enter on “Actions”, “Add” and “Deny Override” controls, Screen Reader is not reading anything about the revealed content.

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#76]
**Environment Details:**
URL: https://manage.iis.net/webserver/modules
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
**WCAG ID Violated:** WCAG1.3.1

**GitHubTags:** #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Screen Reader; #C+E-H; #.NET Websites; #Edge; #Chrome; #Narrator; #NVDA; #WCAG1.3.1

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Server and press enter
5.	Tab till Modules and press enter
6.	Start Screen Reader (Narrator/NVDA).
7.	Validate that screen reader is reading the revealed content or not

**Actual Result:**
when we press enter on “Actions”, “Add” and “Deny Overide” buttons, dropdown is getting expanded, editable content is getting revealed but narrator/nvda is not reading anything about the revealed content.

**Expected Result:**
when we press enter on “Actions”, “Add” and “Deny Overide” buttons, narrator/nvda should read about revealed content

**User Impact:**
Users would not be able to understand whether any content got revealed or not and also the action performed by those controls.

**Comment by v-supupp** [comment]
**Attachment video for your reference:**
[Bug 76_Narrator_Revealed Content.pptx](https://github.com/Microsoft/IIS.WebManager/files/2318148/Bug.76_Narrator_Revealed.Content.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
Fixed

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
